cob/cob.c

```diff
@@ -521,7 +707,7 @@ M0_INTERNAL int m0_cob_domain_credit_add(struct m0_cob_domain          *dom,
 		return M0_ERR(-ENOMEM);
 	m0_be_0type_add_credit(bedom, &m0_be_cob0, cdid_str, &data, cred);
 	M0_BE_ALLOC_CREDIT_PTR(dom, seg, cred);
-	m0_be_btree_create_credit(&dummy, 5 /* XXX */, cred);
+	m0_be_btree_create_credit(&dummy, 6 /* XXX */, cred);
```

Comment:
What is this dummy reference, and hardcoded value '6'.

Reply:
1. The dummy reference here is set to the segments btree using code `struct m0_be_btree   dummy = { .bb_seg = seg };` and therefore we are allowing credits to create btree.
2. Hardcoded value '6' is the number of credits used when btree's get created in cob domain. Now we have 6 btree's after adding `cd_bytecount` tree in cob domain, so we are using hardcoded value '6'.